ros2node/ros2node/api/__init__.py

```diff
@@ -145,6 +156,45 @@ def get_action_client_info(*, node, remote_node_name, include_hidden=False):
         for n, t in names_and_types if include_hidden or not _is_hidden_name(n)]
 
 
+def call_log_level_set(node, node_name, level):
+    """
+    Set the log level of the specified node using the SetLoggerLevels ROS service.
+
+    :param node: The rclpy node to use as the client
+    :param node_name: The full name of the target node
+    :param level: The log level as a string (e.g., 'DEBUG', 'INFO', 'WARN', 'ERROR', 'FATAL')
+    """
+    # Prepare the service name for the target node
+    service_name = f'{node_name}/set_logger_levels'
+    client = node.create_client(SetLoggerLevels, service_name)
+
+    if not client.service_is_ready():
+        raise RuntimeError(f'Service {service_name} not ready')
```

Comment:
Just a comment about this part specifically:

As Tomoya said, the node will not have this service if its `enable_logger_service` node option isn't set to `true`: https://docs.ros.org/en/rolling/Tutorials/Demos/Logging-and-logger-configuration.html#logger-level-configuration-externally. It is `false` by default: https://github.com/ros2/rclcpp/blob/84c6fb1cfc945521680a0e1fccf5b32237acbcc3/rclcpp/include/rclcpp/node_options.hpp#L452

Instead of just throwing an error with a generic `f'Service {service_name} not ready'` message, it should let the user know that the service may not exist if the node doesn't have the `enable_logger_service` option enabled. This way the user knows exactly what to change in their code. Something like:

```suggestion
    # Prepare the service name for the target node
    service_name = f'{node_name}/set_logger_levels'
    client = node.create_client(SetLoggerLevels, service_name)

    if not client.service_is_ready():
        raise RuntimeError(
            f"Service '{service_name}' is not available. This service is only available if the "
            f"'enable_logger_service' node option is enabled for node '{node_name}'."
        )
```

Reply:
ah! I missed this from your earlier comment:

> Maybe it could be logged more clearly like: "Service not available. Are the logging services enabled?"

Then yes, definitely!

Reply:
Updated log line

Reply:
I think this should explicitly mention the "'enable_logger_service' node option" not just "Are the logging services enabled?"

Users probably have no idea how to enable logging services. It's much more useful to tell them exactly what needs to change: enable the `enable_logger_service` node option